dags/blocksec_plugin/ricochet_streamer_close_operator.py

```diff
@@ -1,57 +1,25 @@
-from airflow.models.baseoperator import BaseOperator
 from airflow.utils.decorators import apply_defaults
-from blocksec_plugin.web3_hook import Web3Hook
-from blocksec_plugin.ethereum_wallet_hook import EthereumWalletHook
+from blocksec_plugin.contract_interaction_operator import ContractInteractionOperator
 from blocksec_plugin.abis import RICOCHET_ABI
```

Comment:
This `RICOCHET_ABI` probaby should be more descriptive, `REX_ABI` is for this, `REX_BANK_ABI` for the bank stuff, `REX_LAUNCHPAD_ABI` for the laucnhpad